ixmp/backend/jdbc.py

```diff
@@ -603,13 +642,10 @@ def init_item(self, s, type, name, idx_sets, idx_names):
         try:
             func(name, idx_sets, idx_names)
         except jpype.JException as e:
-            e = str(e)
-            if 'This Scenario cannot be edited' in e:
-                raise RuntimeError(e)
-            elif 'already exists' in e:
-                raise ValueError('{!r} already exists'.format(name))
+            if 'already exists' in e.args[0]:
```

Comment:
Is it IxException? Maybe change exception handling in java?

Reply:
Same response as https://github.com/iiasa/ixmp/pull/289#discussion_r397454945.